tests/test_presentation.py

```diff
@@ -473,6 +500,11 @@ def test_link_object_role(self):
         assert element.text == linking_object.object_role
         assert has_premis_namespace(element)
 
+    def test_validate_eventxml(self, premis_schema):
+        evt = factories.EventFactory()
```

Comment:
I think just using the full name `event` would be better for consistency with previous tests and for readability.